lib/nexpose/scan_template.rb

```diff
@@ -191,82 +191,82 @@ def enable_ip_stack_fingerprinting=(enable)
       end
     end
 
-    # Enable/diisable ICMP device discovery
-    # @param [Boolean] enable or disable ICMP device discovery
-    def enable_icmp_discovery=(enable)
+    # Enable/disable ICMP device discovery
+    # @param [Boolean] enable Enable or disable ICMP device discovery
+    def enable_icmp_device_discovery=(enable)
```

Comment:
Renaming from 0.9.7? Should another version be published immediately to reduce breakage?